docs/troubleshooting.md

```diff
@@ -190,3 +225,194 @@ __Troubleshooting__:
 If [ExternalDNS is updating DNS zone records](#externaldns-is-not-updating-dns-zone-records) and the [DNS glue records are configured](#dns-glue-records-are-not-configured) correctly, you need to wait for the DNS propagation to complete.
 
 [whatsmydns.net](https://www.whatsmydns.net) is a DNS propagation checker that can give you a rough idea of the DNS propagation status of your DNS records.
+
+## Troubleshooting BKPR Ingress
```

Comment:
I think I may have misunderstood the structure of this document but it looks like there is a section with the same title further up? What's the relationship between the two, or is this just a GitHub diff view issue?

Reply:
No, it was my mistake when merging/rebasing on top of the previous PR from Sameer. It should be fixed by now.